wiki/Skinning_Tutorial/en.md

```diff
@@ -24,13 +24,13 @@ Once you have your own skinning folder in the `Skins/` folder, you will first ne
 
 ---
 
-The next thing you will need is a theme or an idea. You should have one in mind to create an objective for what you want your skin to look like. This is also to help yourself when you need to gather and/or create assets.
+The most important thing you will need is a theme or an idea. You should have one in mind to create an objective for what you want your skin to look like. This is also to help yourself when you need to gather and/or create assets.
```

Comment:
Theme is rather not so important as skins are mostly aimed for tailoring the user experience for the author.